Realm/Realm.Weaver/Analytics/Analytics.cs

```diff
@@ -209,33 +218,34 @@ internal void AnalyzeUserAssembly(ModuleDefinition module)
 
-                _realmEnvMetrics["UserId"] = AnonymizedUserID;
-                _realmEnvMetrics["ProjectId"] = SHA256Hash(Encoding.UTF8.GetBytes(module.Name));
-                _realmEnvMetrics["RealmSdk"] = ".NET";
-                _realmEnvMetrics["Language"] = "C#";
-                _realmEnvMetrics["HostOsType"] = ConvertPlatformIdOsToMetricVersion(Environment.OSVersion.Platform);
-                _realmEnvMetrics["HostOsVersion"] = Environment.OSVersion.Version.ToString();
-                _realmEnvMetrics["HostCpuArch"] = GetHostCpuArchitecture;
-                _realmEnvMetrics["TargetOsType"] = _config.TargetOSName;
-                _realmEnvMetrics["TargetCpuArch"] = GetTargetCpuArchitecture(module);
-                _realmEnvMetrics["FrameworkUsedInConjunction"] = frameworkInfo.Name;
-                _realmEnvMetrics["FrameworkUsedInConjunctionVersion"] = frameworkInfo.Version;
-                _realmEnvMetrics["LanguageVersion"] = GetLanguageVersion(module, _config.TargetFramework);
-                _realmEnvMetrics["RealmSdkVersion"] = module.FindReference("Realm").Version.ToString();
-                _realmEnvMetrics["CoreVersion"] = "FILL ME";
-                _realmEnvMetrics["SdkInstallationMethod"] = "FILL ME";
-                _realmEnvMetrics["IdeUsed"] = "MSBuild";
-                _realmEnvMetrics["NetFramework"] = _config.TargetFramework;
-                _realmEnvMetrics["NetFrameworkVersion"] = _config.TargetFrameworkVersion;
+                _realmEnvMetrics[UserEnvironment.UserId] = AnonymizedUserID;
+                _realmEnvMetrics[UserEnvironment.ProjectId] = SHA256Hash(Encoding.UTF8.GetBytes(module.Name));
+                _realmEnvMetrics[UserEnvironment.RealmSdk] = ".NET";
+                _realmEnvMetrics[UserEnvironment.Language] = "C#";
+                _realmEnvMetrics[UserEnvironment.HostOsType] = GetHostOsName();
+                _realmEnvMetrics[UserEnvironment.HostOsVersion] = Environment.OSVersion.Version.ToString();
+                _realmEnvMetrics[UserEnvironment.HostCpuArch] = GetHostCpuArchitecture;
+                _realmEnvMetrics[UserEnvironment.TargetOsType] = _config.TargetOSName;
+                _realmEnvMetrics[UserEnvironment.TargetCpuArch] = GetTargetCpuArchitecture(module);
+                _realmEnvMetrics[UserEnvironment.FrameworkUsedInConjunction] = frameworkInfo.Name;
+                _realmEnvMetrics[UserEnvironment.FrameworkUsedInConjunctionVersion] = frameworkInfo.Version;
+                _realmEnvMetrics[UserEnvironment.LanguageVersion] = GetLanguageVersion(_config.TargetFramework);
+                _realmEnvMetrics[UserEnvironment.RealmSdkVersion] = module.FindReference("Realm").Version.ToString();
+                _realmEnvMetrics[UserEnvironment.CoreVersion] = "FILL ME";
+                _realmEnvMetrics[UserEnvironment.SdkInstallationMethod] = "FILL ME";
+                _realmEnvMetrics[UserEnvironment.IdeUsed] = "FILL ME";
```

Comment:
I thought we'll report the Unity editor for the IDE unless we're capturing that information somewhere else? Also, since it's not clear how we'll capture the msbuild version, I thought it's better to leave it blank for now.

Reply:
> I thought we'll report the Unity editor for the IDE
No no, we're reporting Unity under frameworks as it's "the framework" that Realm is being used with. Same as we do for xamarin, maui etc.
Then I'd just leave the field empty. I changed to this "empty" instead of "FILL ME" in my PR